js/base/Exchange.js

```diff
@@ -2063,19 +2063,20 @@ module.exports = class Exchange {
         let value = this.safeString2 (params, optionName, defaultOptionName);
         if (value !== undefined) {
             params = this.omit (params, [ optionName, defaultOptionName ]);
-        }
-        if (value === undefined) {
-            // check if exchange-wide method options contain the key
+        } else {
+            // check if exchange has properties for this method
             const exchangeWideMethodOptions = this.safeValue (this.options, methodName);
             if (exchangeWideMethodOptions !== undefined) {
+                // check if the option is defined in this method's props
                 value = this.safeString2 (exchangeWideMethodOptions, optionName, defaultOptionName);
             }
+            if (value === undefined) {
+                // if it's still undefined, check if global exchange-wide option exists
+                value = this.safeString2 (this.options, optionName, defaultOptionName);
+            }
+            // if it's still undefined, use the default value
+            value = (value !== undefined) ? value : defaultValue;
         }
-        if (value === undefined) {
```

Comment:
this check here was not perfect, because `if (value === undefined)` only could happen inside 2067th line clause, so, we have to move this block inside there for efficiency.